server/routerlicious/packages/services/src/deltaManager.ts

```diff
@@ -93,6 +95,36 @@ export class DeltaManager implements IDeltaService {
 			};
 		};
 
+		const refreshTokenIfNeeded = async (authorizationHeader: RawAxiosRequestHeaders) => {
+			if (
+				authorizationHeader.Authorization &&
+				typeof authorizationHeader.Authorization === "string"
+			) {
+				const currentAccessToken = extractTokenFromHeader(
+					authorizationHeader.Authorization,
+				);
+				if (isTokenValid(currentAccessToken)) {
+					Lumberjack.info(`Token is still valid for deltaManager`, {
```

Comment:
I wonder if this log could get too noisy? Maybe replace it with verbose if that is the case.